website/content/_index.md

```diff
@@ -16,23 +16,23 @@ explains what that implies.
 
 ## Why?
 
-Kubernetes does not offer an implementation of network load-balancers
+Kubernetes does not offer an implementation of network load balancers
 ([Services of type LoadBalancer](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/))
-for bare metal clusters. The implementations of Network LB that
+for bare-metal clusters. The implementations of network load balancer that
```

Comment:
I think this should be plural?

```suggestion
for bare-metal clusters. The implementations of network load balancers that
```

Reply:
That's def an oops.  Thank you for noticing and pointing it out.